utils/appliance.py

```diff
@@ -329,7 +331,33 @@ def remove_rhev_direct_lun_disk(self, log_callback=None):
             raise ApplianceException(msg)
 
 
-class IPAppliance(object):
+class SproutAppliance(Appliance):
+    """Appliance provisioned through sprout
+
+    Overrides the `destroy` method to get rid of the appliance properly.
+    """
+
+    pretty_attrs = ['sprout_id'] + Appliance.pretty_attrs
+
+    def __init__(self, sprout_id, **kwargs):
+        self.sprout_id = sprout_id
+        # Check that it exists in sprout
+        app_data = self.cli.appliance_data(sprout_id)
+        # Get data to pass into Appliance obj
+        kwargs['provider_name'] = app_data['provider']
+        kwargs['vm_name'] = app_data['name']
+        super(SproutAppliance, self).__init__(**kwargs)
+
+    def destroy(self):
+        """Kill through sprout"""
+        self.cli.destroy_appliance(self.sprout_id)
+
+    @lazycache
+    def cli(self):
```

Comment:
I might go even further and go with `sprout_client`, but I'm wondering if it makes sense to make the sprout-related attributes and methods private, since as far as I can tell they're only used internally to `SproutAppliance`.